### PR TITLE
Sample Notification Pop-up

### PR DESCRIPTION
**Pull request by DamianSuess**
Feature Includes:

* Added Notification Pop-up Sample 
* Spelling corrections
* Added XamlStyler config - _Ensures user's personal settings don't override project styling_

### CODE REVIEW

**Comment by DamianSuess** [comment]
Ready for review

File: samples/SampleMvvmApp/Services/NotifictionService.cs [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+        /// <param name="title">Title.</param>
+        /// <param name="message">Message.</param>
+        /// <param name="onClick">Optional OnClick action.</param>
+        public void Show(string title, string message, Action? onClick = null)
+        {
+            if (_notificationManager is { } nm)
+            {
+                // Alt method:
+                //// RxApp.MainThreadScheduler.Schedule(() =>
+                Dispatcher.UIThread.InvokeAsync(() =>
```
Comment (OmidID) [inline samples/SampleMvvmApp/Services/NotifictionService.cs:47]: @DamianSuess Why ThreadSafe should be check in the service layer? Is it call by app directly? then should be manage by app

Comment (DamianSuess) [reply samples/SampleMvvmApp/Services/NotifictionService.cs:47]: Good call 👍

Comment (DamianSuess) [reply samples/SampleMvvmApp/Services/NotifictionService.cs:47]: @OmidID thanks again for taking the time. In this instance, we didn't need the UI Thread call anyways.

Just a heads up, I'm gonna have a couple more PRs coming up in preparation for the Avalonia 0.11 releases

**Pull request merged**